WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserAdapter.java

```diff
@@ -65,6 +68,7 @@ public View newView(Context context, Cursor cursor, ViewGroup parent) {
 
         ThemeViewHolder themeViewHolder = new ThemeViewHolder(view);
         view.setTag(themeViewHolder);
+        mViewWidth = parent.getWidth();
```

Comment:
Or another solution: use the max(width / numberOfColumns portrait, width / numberOfColumns landscape)- so we won't download a second copy of the image when we rotate the screen